virtool/groups/data.py

```diff
@@ -49,10 +50,17 @@ async def create(self, group_id: str) -> VTGroup:
         :return: the group
         """
         try:
+            user_list = []
+            async for user in self._db.users.find():
+                if group_id in user["groups"]:
+                    user_minimal = get_user_minimal(user)
+                    user_list.append(user_minimal)
+
             document = await self._db.groups.insert_one(
                 {
                     "_id": group_id,
                     "permissions": generate_base_permissions(),
+                    "users": user_list,
```

Comment:
Don't include `users` in group document. The response field is derived from the users collection.